qcfractal/queue/managers.py

```diff
@@ -448,11 +497,24 @@ def update(self, new_tasks: bool = True, allow_shutdown=True) -> bool:
 
             self.active -= n_result
             for key, result in results.items():
+                wall_time_seconds = 0
                 if result.success:
                     n_success += 1
+                    try:
```

Comment:
Would recommend `if hasattr(result.provenance 'wall_time')`.

Reply:
That's a cleaner construct, I'll change that